rioxarray/_io.py

```diff
@@ -1262,6 +1262,12 @@ def open_rasterio(
 
     if chunks is not None:
         result = _prepare_dask(result, riods, filename, chunks)
+    else:
+        result.encoding["preferred_chunks"] = {
+            result.rio.y_dim: riods.block_shapes[0][0],
```

Comment:
I just remembered this issue related to the default chunk size: #253

Reply:
Relevant comment: https://github.com/corteva/rioxarray/issues/253#issuecomment-1226709359

> There are many considerations for optimal chunks (see [dask blog](https://blog.dask.org/2021/11/02/choosing-dask-chunk-sizes)), but a good default size could be determined by ~100Mb and a multiple of 256x256?

Reply:
I am wondering if you simply multiplied each dimension by 10 if that would be more optimal.

Something like this:
```python
riods.block_shapes[0][0] * 10 if (riods.block_shapes[0][0] < 1000) and (riods.block_shapes[0][0] * 10 < riods.height) else riods.block_shapes[0][0]
```

Reply:
I don't understand why the chunk size needs to be updated? What we are looking for here with `preferred_chunks` is the size of the chunk on disk. Xarray/dask will then use that to compute the optimal size (multiple), so I don't think we need to do anything here.

Reply:
Sounds good :+1: